controllers/provider-alicloud/charts/internal/seed-controlplane/charts/alicloud-cloud-controller-manager/templates/configmap-monitoring.yaml

```
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloud-controller-manager-monitoring-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    extensions.gardener.cloud/configuration: monitoring
+data:
+  scrape_config: |
+    scrape_configs:
+    - job_name: cloud-controller-manager
+      honor_labels: false
+      kubernetes_sd_configs:
+      - role: endpoints
+        namespaces:
+          names: {{ .Release.Namespace }}
+      relabel_configs:
+      - source_labels:
+        - __meta_kubernetes_service_name
+        - __meta_kubernetes_endpoint_port_name
+        action: keep
+        regex: cloud-controller-manager;metrics
+      # common metrics
+      - action: labelmap
+        regex: __meta_kubernetes_service_label_(.+)
+      - source_labels: [ __meta_kubernetes_pod_name ]
+        target_label: pod
+      metric_relabel_configs:
+      - source_labels: [ __name__ ]
+        regex: ^(rest_client_requests_total|process_max_fds|process_open_fds)$
+        action: keep
+
+  alerting_rulse: |
```

```suggestion
  alerting_rules: |
```

(everywhere else as well)